src/redux/books/books.js

```diff
@@ -0,0 +1,38 @@
+// Actions
+const ADD_BOOK = 'ADD_BOOK';
+const REMOVE_BOOK = 'REMOVE_BOOK';
```

Comment:
- [ ] Kindly make the value of each action type to follow the [ducks pattern](https://github.com/erikras/ducks-modular-redux), so it will be something like:  `const ADD_BOOK = bookstore/books/ADD_BOOK `and `const REMOVE_BOOK = bookstore/books/REMOVE_BOOK`